pkg/client/src/app/queries/imports.ts

```diff
@@ -42,16 +41,16 @@ export const useFetchImportSummaries = () => {
   const [importSummaries, setImportSummaries] = useState<
     ApplicationImportSummary[]
   >([]);
-  const { isLoading, error, refetch } = useQuery(ImportsQueryKey, () =>
-    getApplicationImportSummary()
-      .then(({ data }) => {
-        setImportSummaries(data);
-        return data;
-      })
-      .catch((error) => {
-        console.log("error, ", error);
-        return error;
-      })
+  const { isLoading, error, refetch } = useQuery(
+    "importsummaries",
```

Comment:
Looking good! One small thing -  We should define this as a constant and export it as we do in the other queries. Thanks!

Reply:
I'm more than happy to do that when the key is reused which is not the case anymore.

Reply:
If we ever need to reference the imports query, it would be nice to have it available. Seems silly to remove.